docs/CHANGELOG.rst

```diff
@@ -15,11 +15,13 @@ Added
 * Add Resolwe API to test url configuration
 * Add TransactionBioProcessTestCase to support writing tests for
   processes which use resdk
+* Add ``genialis/chipseq`` docker image
```

Comment:
I think we should use `resolwebio/chipseq`. We use `genialis/iclip` because the corresponding pipelines repository is not open source.

Also, we already use [`resolwebio/archive-samples`](https://hub.docker.com/r/resolwebio/archive-samples/).